src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -928,32 +928,89 @@ void EnIk_Update(Actor* thisx, PlayState* play2) {
     EnIk_UpdateArmor(this, play);
 }
 
-s8 D_8092C1A8[] = {
-    -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 0,  -1, -1, -1,
-    -1, -1, -1, -1, -1, -1, 3,  5,  4,  6,  1,  2,  -1, -1, 0,  0,
+static s8 sLimbToBodyParts1[IRON_KNUCKLE_LIMB_MAX] = {
+    -1, // IRON_KNUCKLE_LIMB_NONE
+    -1, // IRON_KNUCKLE_LIMB_ROOT
+    -1, // IRON_KNUCKLE_LIMB_TASSET_CENTER
+    -1, // IRON_KNUCKLE_LIMB_TASSET_RIGHT
+    -1, // IRON_KNUCKLE_LIMB_RIGHT_LEG
+    -1, // IRON_KNUCKLE_LIMB_RIGHT_FOOT
+    -1, // IRON_KNUCKLE_LIMB_TASSET_LEFT
+    -1, // IRON_KNUCKLE_LIMB_LEFT_LEG
+    -1, // IRON_KNUCKLE_LIMB_LEFT_FOOT
+    -1, // IRON_KNUCKLE_LIMB_UPPER_BODY_ROOT
+    -1, // IRON_KNUCKLE_LIMB_UNK_ROOT
+    -1, // IRON_KNUCKLE_LIMB_HEAD_ROOT
+    0,  // IRON_KNUCKLE_LIMB_HELMET_ARMOR
+    -1, // IRON_KNUCKLE_LIMB_HEAD
+    -1, // IRON_KNUCKLE_LIMB_RIGHT_UPPER_ARM
+    -1, // IRON_KNUCKLE_LIMB_RIGHT_FOREARM
+    -1, // IRON_KNUCKLE_LIMB_AXE_ROOT
+    -1, // IRON_KNUCKLE_LIMB_AXE
+    -1, // IRON_KNUCKLE_LIMB_RIGHT_HAND
+    -1, // IRON_KNUCKLE_LIMB_LEFT_UPPER_ARM
+    -1, // IRON_KNUCKLE_LIMB_LEFT_FOREARM
+    -1, // IRON_KNUCKLE_LIMB_LEFT_HAND
+    3,  // IRON_KNUCKLE_LIMB_UPPER_LEFT_PAULDRON
+    5,  // IRON_KNUCKLE_LIMB_LOWER_LEFT_PAULDRON
+    4,  // IRON_KNUCKLE_LIMB_UPPER_RIGHT_PAULDRON
+    6,  // IRON_KNUCKLE_LIMB_LOWER_RIGHT_PAULDRON
+    1,  // IRON_KNUCKLE_LIMB_CHEST_ARMOR_FRONT
+    2,  // IRON_KNUCKLE_LIMB_CHEST_ARMOR_BACK
```

Comment:
Some actors like the iron knuckle have two sets of "body parts", so I'm not sure if I should just make two sets of body parts enums?

Reply:
I would probably say yeah, two enums.